### DISCUSSION TITLE
How to configure Pi-Hole and Adguard?

### DISCUSSION THREAD

**Discussion by xXxKIRSHxXx** [IceWhaleTech/CasaOS#619] [Q&A]
I don't work with technology and computing but I am interested in the subject. 

I think Casaos is very good, but I think you could improve the small details or leave information on how to configure the applications inside Casaos itself. I, for example, tried to configure Pi-Hole and Adguard for several days but I couldn't even watching several tutorials on the internet :(

P.s. Sorry for not knowing how to write correctly in English.

**Comment by zhanghengxin** [comment]
ithink your question belongs to the community building

**Comment by zhanghengxin** [comment]
so i recommend you join the discord community

here is our link 
https://discord.gg/f3j3jANf
